tests/unit_tests/constraint/test_pcalg.py

```diff
@@ -0,0 +1,255 @@
+import networkx as nx
+import numpy as np
+import pandas as pd
+import pytest
+from numpy.testing import assert_array_equal
+from pywhy_graphs import ADMG, CPDAG
+
+from dodiscover import Context
+from dodiscover.ci import GSquareCITest, Oracle
+from dodiscover.constraint import PC
+from dodiscover.constraint.utils import dummy_sample
+from dodiscover.metrics import confusion_matrix_networks
+from dodiscover.testdata.testdata import bin_data, dis_data
+
+
+@pytest.mark.parametrize(
```

Comment:
@darthtrevino any thoughts on this testing approach, relative to unittest?

Reply:
pytest is significantly better from my exp :p